src/content/en/2020/seo.md

```diff
@@ -83,620 +83,714 @@ Response Code | All Devices | Mobile | Desktop
-
-More information on the Lighthouse performance testing results and score details is available in the Performance section.
-
-#### Web vitals and other CrUX Metrics per Device
-
-Desktop continues to be the more performant platform for users despite more users on mobile devices. 33.13% of websites scored 'Good' Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.
+Desktop continues to be the more performant platform for users despite more users on mobile devices. 33.13% of websites scored _Good_ Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.
 
+{# TODO(analysts): Please double check the following two sql_files, as these metrics are related to Lighthouse. #}
 {{ figure_markup(
   image="seo-good-core-web-vitals-score-per-device.png",
-  caption="Good Core Web Vitals score per device",
-  description="Bar chart showing the percent of websites with a good core web vitals score per device. 33.13% of websites scored 'Good' Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.",
+  caption="Percent of websites passing the Core Web Vitals assessment per device.",
+  description="Bar chart showing the percent of websites with a good core web vitals score per device. 33.13% of websites scored _Good_ Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=1601210449&format=interactive",
   sheets_gid="996380787",
   sql_file="lighthouse.sql"
-  )
-}}
+) }}
 
-#### Web vitals and other CrUX Metrics per Country
+#### Core Web Vitals experiences per country
 
 A user's physical location impacts performance perception as their locally available telecom infrastructure, network bandwidth capacity, and the cost of data create unique loading conditions.
 
-Users located in the United States recorded the largest number of websites with 'Good' Core Web Vitals experiences but these 'Good' accounted for only 31.88% of all websites. At 56.63%, China recorded the highest percentage of 'Good' Core Web Vital experiences. The portion of websites each country represents in the Chrome User Experience Report data set is worth noting as 1,622,765 total origin records generated from users in the United States dwarfs the 21,270 origins requested from users in China.
+{# TODO(authors): China isn't included in this chart because it has a smaller number of websites. Is it worth discussing Korea instead so readers can see how it compares? #}
```

Comment:
@aleyda

Reply:
@rviscomi thanks Rick! @fellowhuman1101 could you please also take a look at this when you check the changes, to mention something about Korea (with the highest percentage of sites with "good" CWV).

Reply:
```suggestion
{# TODO(authors): China isn't included in this chart because it has a smaller number of websites. Is it worth discussing Korea instead so readers can see how it compares? #}

Users located in the United States recorded the largest number of websites with 'Good' Core Web Vitals experiences despite only 32% of sites earning the passing grade. Republic of Korea recorded the highest percentage of 'Good' Core Web Vital experiences at 52%. The relative portion of total websites requested by each country is worth noting. Users in United States generated 8X the total origin requests as generated by Republic of Korea users.
```